packages/reference/src/common/EntityStore.tsx

```diff
@@ -78,7 +78,7 @@ type FetchServiceOptions<
   TQueryFnData = unknown,
   TError = unknown,
   TData = TQueryFnData,
-  TQueryKey extends QueryKey = QueryKey
+  TQueryKey extends QueryKey = QueryKey,
```

Comment:
Why is this file in this PR?

Reply:
Oh i think i accidentially opened it and it got prettied. i will revert the changes :)